--- nativeLibs/Pods/AzureCommunicationCalling/AzureCommunicationCalling.framework/Headers/AzureCommunicationCalling.h	2021-05-13 22:27:29.000000000 +0200
+++ nativeLibs/Pods/AzureCommunicationCalling/AzureCommunicationCalling.framework/Headers/AzureCommunicationCalling_new.h	2021-05-13 22:32:39.000000000 +0200
@@ -3,7 +3,7 @@
 
 #import <Foundation/Foundation.h>
 
-#import <AzureCommunicationCommon/AzureCommunicationCommon-Swift.h>
+#import "../../../AzureCommunicationCommon/AzureCommunicationCommon.framework/Headers/AzureCommunicationCommon-Swift.h"
 #import <CallKit/CallKit.h>
 #import "ACSVideoStreamRenderer.h"
 #import "ACSVideoStreamRendererView.h"
