--- nativeLibs/Pods/AzureCommunicationCalling/AzureCommunicationCalling.framework/Headers/AzureCommunicationCalling.h	2020-11-19 15:42:45.000000000 +0100
+++ nativeLibs/Pods/AzureCommunicationCalling/AzureCommunicationCalling.framework/Headers/AzureCommunicationCalling.h	2020-11-19 16:30:27.000000000 +0100
@@ -3,8 +3,7 @@
 
 #import <Foundation/Foundation.h>
 
-#import <AzureCommunication/AzureCommunication-Swift.h>
-//#import <AzureCore/AzureCore-Swift.h>
+#import "../../../AzureCommunication/AzureCommunication.framework/Headers/AzureCommunication-Swift.h"
 #import "ACSRenderer.h"
 #import "ACSRendererView.h"
 #import "ACSStreamSize.h"
