--- nativeLibs/Pods/AzureCommunicationCalling/AzureCommunicationCalling.framework/Headers/AzureCommunicationCalling.h   2020-11-16 05:57:35.000000000 -0800
+++ nativeLibs/Pods/AzureCommunicationCalling/AzureCommunicationCalling.framework/Headers/AzureCommunicationCalling.h 2020-11-16 06:22:17.000000000 -0800@@ -3,8 +3,8 @@
@@ -3,8 +3,8 @@

 #import <Foundation/Foundation.h>
 
-#import <AzureCommunication/AzureCommunication-Swift.h>
-#import <AzureCore/AzureCore-Swift.h>
+#import "../../../AzureCommunication/AzureCommunication.framework/Headers/AzureCommunication-Swift.h"
+#import "../../../AzureCore/AzureCore.framework/Headers/AzureCore-Swift.h"
 #import "ACSRenderer.h"
 #import "ACSRendererView.h"
 #import "ACSStreamSize.h"
